### ISSUE TITLE
[ ENH ] customizing log path

### ISSUE THREAD

**Issue by pyporn-san** [hentai-chan/hentai#86]
#### Is your feature request related to a problem? Please describe

The problem happens when I try to use the library in a system that does not have write permission to /var/log
the issue happens when assigning the file handler for logger

#### Describe the solution you'd like

Add the ability to change the logging file location or disable logging completely

**Comment by hentai-chan** [comment]
Thanks for filing this issue. I'll address this problem in the next version of this module. In the meantime, there're two things that you can do:

1. If you can elevate file permission, create the logfile and change ownership to the active user 
2. Downgrade your installation to version 3.2.3 until the next release

---

## 1. Elevating File Ownership

Of course this step also preemptively requires sudo privileges which you might or might not have depending on which machine you intend to run your python project.

```bash
cd //
cd var/log
sudo touch hentai.log
sudo chown {{user}} hentai.log
```

## 2. Downgrading Installation

Version `3.2.4` only added a rudimentary [CLI to this library](https://github.com/hentai-chan/hentai/releases/tag/v3.2.4), so if you can live without that I recommend that you downgrade to `3.2.3` until a new release comes out.

```bash
pip install hentai==3.2.3
```

This version created `hentai/hentai.log` in the user's home directory. I didn't want to clutter my home directory, hence the change. I'll try to find a solution that doesn't require elevated permissions.

**Comment by hentai-chan** [comment]
Resolved with version 3.2.5. To update your installation, run 
```bash
pip install --upgrade hentai
```
